src/op_mode/scan_ports_popular.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+
+def scan_popular_ports(host):
+    # List of popular ports to scan
+    popular_ports = [
```

Comment:
Popular according to whom? I see no reason to have a special case for this. 21 (FTP) and 23 (telnet) are quite rare now, but it's beside the point. People who want specific ports can specify a list in the command argument.